xml/ntp_banners.xml

```diff
@@ -419,50 +419,382 @@ NTP "banners", taken from a readvar response
+    <param pos="0" name="service.product" value="NTP"/>
+    <param pos="1" name="service.version"/>
+    <param pos="2" name="os.arch"/>
+    <param pos="3" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="^.*version=&quot;ntpd ([^ ]+)[^&quot;]+&quot;,.*processor=&quot;([^ ]+)&quot;,.*system=&quot;SunOS/?5.([789]|10)&quot;" flags="REG_DOT_NEWLINE,REG_ICASE">
+    <description>ntpd running on Solaris 7-Solaris 10 (SunOS/5.7 - SunOS/5.10) </description>
+    <example service.version="4.2.0@1.1161-r" os.arch="sun4u" os.version="7">
+      version="ntpd 4.2.0@1.1161-r Wed Apr 20 11:28:05 EDT 2005 (1)",
+      processor="sun4u", system="SunOS/5.7", leap=00, stratum=2,
+    </example>
+    <example service.version="4.2.0@1.1161-r" os.arch="sun4u" os.version="10">
+      version="ntpd 4.2.0@1.1161-r Wed Apr 20 11:28:05 EDT 2005 (1)",
+      processor="sun4u", system="SunOS/5.10", leap=00, stratum=2,
+    </example>
+    <example service.version="4.2.0@1.1161-r" os.arch="sun4u" os.version="7">
+      version="ntpd 4.2.0@1.1161-r Wed Apr 20 11:28:05 EDT 2005 (1)",
+      processor="sun4u", system="SunOS5.7", leap=00,stratum=2,
+    </example>
+    <example service.version="4.2.0@1.1161-r" os.arch="sun4u" os.version="10">
+      version="ntpd 4.2.0@1.1161-r Wed Apr 20 11:28:05 EDT 2005 (1)",
+      processor="sun4u", system="SunOS5.10", leap=00, stratum=2,
+    </example>
+    <param pos="0" name="os.vendor" value="Sun"/>
+    <param pos="0" name="os.family" value="Solaris"/>
+    <param pos="0" name="os.product" value="Solaris"/>
+    <param pos="0" name="service.family" value="NTP"/>
+    <param pos="0" name="service.product" value="NTP"/>
+    <param pos="1" name="service.version"/>
+    <param pos="2" name="os.arch"/>
+    <param pos="3" name="os.version"/>
+  </fingerprint>
+  <fingerprint pattern="^.*processor=&quot;([^ ]+)&quot;,.*system=&quot;SunOS/?5.0&quot;" flags="REG_DOT_NEWLINE,REG_ICASE">
+   <description>Solaris 2.0 (SunOS/5.0) with no ntp version</description>
+   <example os.arch="sun4m">
+     processor="sun4m", system="SunOS/5.0"
+   </example>
+   <example os.arch="sun4u">
+    processor="sun4u", system="SunOS/5.0"
+   </example>
+   <example os.arch="sun4m">
+     processor="sun4m", system="SunOS5.0"
+   </example>
+   <example os.arch="sun4u">
+     processor="sun4u", system="SunOS5.0"
+   </example>
+   <param pos="0" name="os.vendor" value="Sun"/>
+   <param pos="0" name="os.family" value="Solaris"/>
+   <param pos="0" name="os.product" value="Solaris"/>
+   <param pos="0" name="os.version" value="2.0" />
+   <param pos="1" name="os.arch"/>
+  </fingerprint>
+  <fingerprint pattern="^.*processor=&quot;([^ ]+)&quot;,.*system=&quot;SunOS/?5.1&quot;" flags="REG_DOT_NEWLINE,REG_ICASE">
+   <description>Solaris 2.1 (SunOS/5.1) with no ntp version</description>
+   <example os.arch="sun4m">
+     processor="sun4m", system="SunOS/5.1"
+   </example>
+   <example os.arch="sun4u">
```

Comment:
I agree, I noticed in the ntp_banners.xml before my changes there were examples such as : 

`    <example>
      processor="sun4m", system="SunOS5.6"
    </example>
`

`    <example>
      processor="sun4u", system="SunOS5.6"
    </example>
`

Within the same fingerprint. My thoughts were the same as yours but I thought we maybe tried to list banners seen rather than unique subsets after seeing these, I will adjust this, cheers for the clarification! :)